src/WorkItems/VssConnectionWrapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+
+namespace Microsoft.WorkItems
+{
+    internal class VssConnectionWrapper : IVssConnection
+    {
+        VssConnection connection;
+
+        async public Task ConnectAsync(Uri accountUri, string personalAccessToken)
+        {
+            this.connection = new VssConnection(accountUri, new VssBasicCredential(string.Empty, personalAccessToken));
```

Comment:
>this.connection [](start = 12, length = 15)

Consider doing this in the ctor and making it readonly. (1) That's the pattern you followed in WorkItemTrackingHttpClientWrapper, (2) it makes it easier for you to explain what it does in the comment in your interface definition, and (3) (related to #2) it's a code smell when a method does two things ("Create the connection AND connect").